docs/recipes/Deploying-Modules.md

```diff
@@ -5,83 +5,23 @@
 # Deploying Modules
 
 ## 📖 Table of Contents
+* [Deployment Overview](deployment-overview)
+* [Creating a Module Map](#creating-a-module-map)
+* [Update Module Map Script](#update-module-map-script)
+* [Deploying to Vercel with GitHub Actions](#deploying-to-vercel-with-github-actions)
 
-* [Deploying to Now with GitHub Actions](#deploying-to-now-with-github-actions)
-  * [Creating your Deploy Action](#creating-your-deploy-action)
-* [Create and Update Your Module Map](#create-and-update-your-module-map)
-  * [Creating a Module Map in Now.sh](#creating-a-module-map-in-nowsh)
-  * [Create an Update Module Script](#create-an-update-module-script)
-  * [Update your Deploy Action to Update your Module Map](#update-your-deploy-action-to-update-your-module-map)
+## Deployment Overview
 
-## Deploying to Now with GitHub Actions
+Deploying holocron modules consists on two steps: 
 
-[Now.sh](https://zeit.co/home) allows you to deploy statics such as your modules. We will accomplish this by using [GitHub Actions](https://github.com/features/actions).
+1. Upload the contents of the `build` folder to your CDN of choice.
```

Comment:
i agree on this, and create separate documentation on how to do this and a step by step on uploading to a cdn/using actions to achieve this

Reply:
@JAdshead do you mean changing the name of the entire guide to be "Publising modules" or is this just for the "deployment overview" section?

Reply:
```suggestion
Publishing holocron modules consists on two steps: 

1. Upload the contents of the `build` folder to your CDN of choice. This publishes your module's static assets similar to when you release an npm dependency.
2. Update the [module map](../api/server/Module-Map-Schema.md) to point to the newly published static assets. This will tell One App to consume that published/released module version.
```

Reply:
Renamed to `Publishing modules`